modules/jlink/tests/features/jlink.feature

```diff
@@ -0,0 +1,15 @@
+@ubi9/openjdk-11
+@ubi9/openjdk-17
+@ubi9/openjdk-21
+Feature: Openshift OpenJDK S2I tests (jlink specific)
+Scenario: Ensure jlinked builder is used to build the containerized application image
+      Given s2i build https://github.com/jboss-container-images/openjdk-test-applications from quarkus-quickstarts/getting-started-3.0.1.Final-nos2i
+       | variable            | value        |
+       | S2I_ENABLE_JLINK    | true         |
+       | QUARKUS_PACKAGE_TYPE| uber-jar     |
+      Then run ls /tmp/jre in container and check its output for bin
+
+Scenario: Ensure S2I_ENABLE_JLINK is not set to true
+      Given s2i build https://github.com/jboss-container-images/openjdk-test-applications from quarkus-quickstarts/getting-started-3.0.1.Final-nos2i
+      Then container log should not contain S2I_ENABLE_JLINK=true
+      And run ls /tmp/jre in container and check its output for No such file or directory
```

Comment:
I'm not able to get this test to pass when it should locally. I'm not sure why: stderr output not in container log? case sensitivity? locale sensitivity for the format of the ls error output? I see this:
`      Exception: ("Phrase 'No such file or directory' was not found in the output of running the 'ls /tmp/jre' command", b"ls: cannot access '/tmp/jre': No such file or directory\n", ExecException('Command ls /tmp/jre failed to execute, return code: 2'))
`

Reply:
I've come up with an alternative solution, I'll push it